README.md

```diff
@@ -12,13 +12,19 @@ All remaining requirements are already included in macOS. Patches are welcome fo
 
 ## Usage
 
-```
+```shell
 windows-esd-to-iso ESD_FILE
 ```
 
+  - Or use a one-liner
```

Comment:
Stylistically, let's not do a single bullet point like this, and while I'm not completely opposed to talking about the "curl | sh" pattern as an option, since this could often be a tool people use once and never again, I think it's better explained in a "getting the tool" section preceding "usage".